### PR TITLE
SmallRye GraphQL Documentation update

### PR DESCRIPTION
**Pull request by phillip-kruger**
Add small section w.r.t new batching in SmallRye GraphQL in Documentation

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

File: docs/src/main/asciidoc/microprofile-graphql.adoc [hunk 1]
```diff
@@ -529,6 +529,20 @@ query getFilmHeroes {
 
 The response now includes the heroes of the film.
 
+=== Batching
+
+When you exposing a Collection return like our `getAllFilms`, you might want to use the batch form of the above, to more efficiently fetch
```
Comment (gsmet) [inline docs/src/main/asciidoc/microprofile-graphql.adoc:534]:
```suggestion
When you are exposing a `Collection` return like our `getAllFilms`, you might want to use the batch form of the above, to more efficiently fetch
```

File: docs/src/main/asciidoc/microprofile-graphql.adoc [hunk 2]
```diff
@@ -529,6 +529,20 @@ query getFilmHeroes {
...
+the heroes:
+
+[source,java]
+----
+    public List<List<Hero>> heroes(@Source List<Film> films) { // <1>
+        // Here fetch all hero lists
+    }
+----
+
+<1> Here receive the films as a batch, allowing you the fetch the corresponding heroes.
```
Comment (gsmet) [inline docs/src/main/asciidoc/microprofile-graphql.adoc:544]:
```suggestion
<1> Here receive the films as a batch, allowing you to fetch the corresponding heroes.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Could you make the commit message more than one word?

`Small` is not particularly helpful.

**Comment by phillip-kruger** [comment]
Thanks @gsmet - not sure what happened there, I thought I typed a full sentence...

**Review by gsmet — APPROVED** [review]

**Pull request merged**